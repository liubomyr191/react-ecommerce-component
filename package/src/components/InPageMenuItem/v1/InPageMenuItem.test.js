import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";
import InPageMenuItem from "./InPageMenuItem";

test("InPageMenuItem basic component", () => {
  const component = renderer.create(<InPageMenuItem href="/test/url/" label="Test label" />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("InPageMenuItem with onClick instead of href", () => {
  const onClick = () => {};
  const component = renderer.create(<InPageMenuItem label="Test label" onClick={onClick} />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("InPageMenuItem selected", () => {
  const component = renderer.create(<InPageMenuItem isSelected href="/test/url/" label="Test label" />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
