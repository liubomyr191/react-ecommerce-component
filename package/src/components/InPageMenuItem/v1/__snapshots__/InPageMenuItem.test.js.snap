// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InPageMenuItem basic component 1`] = `"Link(undefined)"`;

exports[`InPageMenuItem selected 1`] = `"Link(undefined)"`;

exports[`InPageMenuItem with onClick instead of href 1`] = `"Link(undefined)"`;
